cyclops-ui/src/components/pages/ModuleDetails.tsx

```diff
@@ -373,6 +373,10 @@ const ModuleDetails = () => {
     let resourceDetails = <p />;
 
     switch (resource.kind) {
+      case "CronJob":
+        resourceDetails = (
+          <CronJob name={resource.name} namespace={resource.namespace} />
+        );
```

Comment:
Can you add `break;` to the end of the case?
```suggestion
        case "CronJob":
        resourceDetails = (
            <CronJob name={resource.name} namespace={resource.namespace} />
        );
        break;
```

Also, can you put CronJob case below Deployment?